<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AppearanceConfig</class>
 <widget class="QWidget" name="AppearanceConfig">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>258</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string/>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="KButtonGroup" name="kbuttongroup">
     <property name="flat">
      <bool>true</bool>
     </property>
     <layout class="QFormLayout" name="formLayout_4">
      <property name="margin">
       <number>0</number>
      </property>
      <item row="1" column="1">
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QRadioButton" name="defaultIconsRadio">
          <property name="text">
           <string>Default</string>
          </property>
          <property name="checked">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QRadioButton" name="customIconsRadio">
          <property name="text">
           <string>Custom</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QRadioButton" name="svgIconsRadio">
          <property name="text">
           <string>SVG</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="label_6">
        <property name="text">
         <string>Icons:</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="iconGroup">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="title">
      <string>Custom Icons</string>
     </property>
     <property name="flat">
      <bool>false</bool>
     </property>
     <property name="checkable">
      <bool>false</bool>
     </property>
     <property name="checked">
      <bool>false</bool>
     </property>
     <layout class="QFormLayout" name="formLayout">
      <item row="2" column="0">
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>Icon for mode 2:</string>
        </property>
        <property name="buddy">
         <cstring>clickedIconButton</cstring>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="KIconButton" name="clickedIconButton">
        <property name="iconSize">
         <number>16</number>
        </property>
       </widget>
      </item>
      <item row="0" column="0">
       <widget class="QLabel" name="label">
        <property name="text">
         <string>Icon for mode 1:</string>
        </property>
        <property name="buddy">
         <cstring>defaultIconButton</cstring>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="KIconButton" name="defaultIconButton">
        <property name="iconSize">
         <number>16</number>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="svgGroup">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="title">
      <string>SVG</string>
     </property>
     <property name="checkable">
      <bool>false</bool>
     </property>
     <layout class="QFormLayout" name="formLayout_2">
      <item row="0" column="0">
       <widget class="QLabel" name="label_3">
        <property name="text">
         <string>SVG File:</string>
        </property>
        <property name="buddy">
         <cstring>svgRequester</cstring>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="label_4">
        <property name="text">
         <string>Element for mode 1:</string>
        </property>
        <property name="buddy">
         <cstring>element1Edit</cstring>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="label_5">
        <property name="text">
         <string>Element for mode 2:</string>
        </property>
        <property name="buddy">
         <cstring>element2Edit</cstring>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="KUrlRequester" name="svgRequester"/>
      </item>
      <item row="1" column="1">
       <widget class="KLineEdit" name="element1Edit"/>
      </item>
      <item row="2" column="1">
       <widget class="KLineEdit" name="element2Edit"/>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>KIconButton</class>
   <extends>QPushButton</extends>
   <header>kicondialog.h</header>
  </customwidget>
  <customwidget>
   <class>KButtonGroup</class>
   <extends>QGroupBox</extends>
   <header>kbuttongroup.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>KLineEdit</class>
   <extends>QLineEdit</extends>
   <header>klineedit.h</header>
  </customwidget>
  <customwidget>
   <class>KUrlRequester</class>
   <extends>QFrame</extends>
   <header>kurlrequester.h</header>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>defaultIconsRadio</tabstop>
  <tabstop>customIconsRadio</tabstop>
  <tabstop>svgIconsRadio</tabstop>
  <tabstop>defaultIconButton</tabstop>
  <tabstop>clickedIconButton</tabstop>
  <tabstop>svgRequester</tabstop>
  <tabstop>element1Edit</tabstop>
  <tabstop>element2Edit</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>svgIconsRadio</sender>
   <signal>toggled(bool)</signal>
   <receiver>svgGroup</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>201</x>
     <y>14</y>
    </hint>
    <hint type="destinationlabel">
     <x>200</x>
     <y>146</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>customIconsRadio</sender>
   <signal>toggled(bool)</signal>
   <receiver>iconGroup</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>136</x>
     <y>15</y>
    </hint>
    <hint type="destinationlabel">
     <x>134</x>
     <y>52</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
